src/migrations/20240221142726-cache-table-for-training-reports.js

```diff
@@ -0,0 +1,74 @@
+const { prepMigration, removeTables } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      await queryInterface.createTable('EventReportPilotNationalCenterUsers', {
```

Comment:
Any thought for just making the table NationalCenterUsers?

Reply:
😞

Reply:
My bad this is a join table